nuxeo-services/nuxeo-platform-web-common/src/main/resources/OSGI-INF/web-request-controller-contrib.xml

```diff
@@ -39,6 +39,10 @@
       <pattern>${org.nuxeo.ecm.contextPath}/waitdialog.*</pattern>
     </filterConfig>
 
+    <filterConfig name="thumbnail" cached="true" cacheTime="0">
```

Comment:
The filter is not only about thumbnails, so could the name be closer to what it does? Maybe "renditionOrBlob"?